noetic/distribution.yaml

```diff
@@ -1992,6 +1992,21 @@ repositories:
       url: https://github.com/ros-gbp/eml-release.git
       version: 1.8.15-7
     status: unmaintained
+  end_effector:
+    doc:
+      type: git
+      url: https://github.com/ADVRHumanoids/ROSEndEffector.git
+      version: master
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/ADVRHumanoids/ROSEndEffector-release.git
+      version: 1.0.5-1
+    source:
+      type: git
+      url: https://github.com/ADVRHumanoids/ROSEndEffector.git
```

Comment:
@torydebra could it be that the `include/ros_end_effector` subdirectory should've been renamed to `include/end_effector` and it wasn't? As it stands, the package doesn't seem to be installing any headers.